backend/src/graphql/model/GdtEntryList.ts

```diff
@@ -16,15 +19,15 @@ export class GdtEntryList {
   @Field(() => String)
   state: string
 
-  @Field(() => Number)
+  @Field(() => Int)
   count: number
 
   @Field(() => [GdtEntry], { nullable: true })
-  gdtEntries?: GdtEntry[]
+  gdtEntries: GdtEntry[] | null
 
-  @Field(() => Number)
+  @Field(() => Int)
   gdtSum: number
 
-  @Field(() => Number)
+  @Field(() => Int)
```

Comment:
What is this?

Reply:
According to our documentation its a float!